spec/blackjack_score_spec.rb

```diff
@@ -1,56 +1,74 @@
 require 'blackjack_score'
 
 describe BlackjackScore do
+  subject(:blackjack_score) { described_class.new }
 
   it "returns 0 if hand is A" do
-    blackjack_score = BlackjackScore.new
-    expect(blackjack_score.get_hand_value("A")).to eq("Enter a valid card")
+    expect(blackjack_score.get_hand_value(["A"])).to eq(-1)
```

Comment:
I am probably missing knowledge about how blackjack is scored, but the test description seems to be in disagreement with the expectation? (says returns 0 but expects -1)